src/OpenTracing/Exceptions/SpanAlreadyFinished.php

```diff
@@ -16,7 +16,11 @@
  */
 final class SpanAlreadyFinished extends DomainException
 {
-    public static function create(Span $span)
+    /**
+     * @param Span $span
+     * @return SpanAlreadyFinished
+     */
+    public static function forSpan(Span $span)
```

Comment:
I see, so the ideal case should be that we suggest to log a warning when the span is finished and we write to it. What do you think @felixfbecker @beberlei?